data/styles/default/include/priorities_4_overlays.prio.txt

```diff
@@ -987,15 +987,15 @@ historic-pillory                                    # icon z16- (also has captio
 historic-tomb                                       # icon z16- (also has caption(optional) z16-)
 historic-wayside_cross                              # icon z16- (also has caption(optional) z16-)
 historic-wayside_shrine                             # icon z16- (also has caption(optional) z17-)
-man_made-cross
+man_made-cross                                      # icon z16-
```

Comment:
would be good to add a caption for man_made=cross

Reply:
a good idea!